engine/table/src/main/java/io/deephaven/engine/table/impl/select/RollingReleaseFilter.java

```diff
@@ -88,7 +92,9 @@ public void setRecomputeListener(RecomputeListener listener) {
 
     @Override
     public RollingReleaseFilter copy() {
-        return new RollingReleaseFilter(workingSize, rollingSize);
+        final RollingReleaseFilter copy = new RollingReleaseFilter(workingSize, rollingSize);
+        copy.addToUpdateGraphProcessor();
```

Comment:
Same comment as on some other code. You can't add to the UGP without a set listener.